### ISSUE TITLE
Lists.GetAllAsync() limits results to 10 lists

### ISSUE THREAD

**Issue by linh-tran-mobosurvey** [brandonseydel/MailChimp.Net#99]
We have more than 10 lists in MailChimp
However, the code below always returns 10 lists. The lists that are not being returned would be returned when we delete some other lists.
```
var mailChimpListsAwaitable = _mailChimpManager.Lists.GetAllAsync().ConfigureAwait(false);
var mailChimpLists = mailChimpListsAwaitable.GetAwaiter().GetResult();
```

**Comment by brandonseydel** [comment]
Can you try using the GetResponseAsync which takes in some query parameters the default is 10 so I will have to go through and add.  

Date: Tue, 20 Sep 2016 08:10:47 -0700

**Comment by linh-tran-mobosurvey** [comment]
Yes. This worked well. I will try to find a number as highest as possible. 

`var listRequestFilter = new ListRequest { Limit = 50};
var mailChimpListsAwaitable = _mailChimpManager.Lists.GetAllAsync(listRequestFilter).ConfigureAwait(false);
var mailChimpLists = mailChimpListsAwaitable.GetAwaiter().GetResult();
`

**Comment by brandonseydel** [comment]
I did this everywhere so if left unspecified there is a global configuration you can set on the mail chimp manager itself.

        public MailChimpManager Configure(MailChimpConfiguration config)
        {
            Limit = config.Limit == 0 ? int.MaxValue : config.Limit;
            this.ApiKey = string.IsNullOrWhiteSpace(config.ApiKey) ? this.ApiKey : config.ApiKey;            
            return this;
        }

Used like so

var mailChimpManager = new MailChimpManager();
mailChimpManager = mailChimpManager.Configure(new MailChimpOptions{


});

or var mailChimpManager = MailChimpManager.Configure(new mail***

pretty vague but you can look over the commit

**Comment by linh-tran-mobosurvey** [comment]
This didn't work. Returned on 10 results
```

var _mailChimpManager= new MailChimpV3.MailChimpManager(apiKey);
_mailChimpManager= (MailChimpV3.MailChimpManager)_mailChimpManager.Configure(new MailChimpV3.MailChimpConfiguration
            {
                Limit = int.MaxValue
            });

var mailChimpListsAwaitable = _mailChimpManager.Lists.GetAllAsync().ConfigureAwait(false);
var mailChimpLists = mailChimpListsAwaitable.GetAwaiter().GetResult();
```

**Comment by brandonseydel** [comment]
Ok let me look whats going on.  that configure method should be exposed also...
